docs/guides/oomkill.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'The "ommkil" gadget'
+weight: 10
+---
+
+The `oomkill` gadget is used to monitor when OOM killer actually kills a process.
+In this guide, we will learn how to use it by running a small `kubernetes` cluster inside `minikube`.
+
+## How to use it?
+
+First, we need to create one pod which will allocate all the memory:
+```bash
+$ kubectl create ns test-ns
+$ kubectl run -n test-ns --image debian:latest test-pod sleep inf
```

Comment:
What about using a memory limit to avoid affecting the host with this experiment? https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#example-1

Reply:
What about this idea? I think running it without limits could crash user's computer....